AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -122,7 +123,7 @@ - (void)didLoad
       textView.backgroundColor = nil;
       textView.opaque = NO;
     }
-    textView.textContainerInset = UIEdgeInsetsZero;
+    textView.textContainerInset = self.textContainerInset;
```

Comment:
The value of the new property will only be recognized on first load. I would recommend extending `setTextContainerInset:` to support value changes after the view has loaded.